File: app-kit/src/main/java/media/pixi/appkit/ui/chat/textinput/TextInputDelegate.java
package media.pixi.appkit.ui.chat.textinput;

import media.pixi.appkit.data.audio.Recording;

public interface TextInputDelegate {

    void showOptions();
    void hideOptions();
    void onSendPressed(String text);
    void startTyping();
    void sendAudio(Recording recording);
    void stopTyping();
    void onKeyboardShow();
    void onKeyboardHide();

}

File: app-kit/src/main/java/media/pixi/appkit/ui/chat/textinput/TextInputDelegate.kt
package media.pixi.appkit.ui.chat.textinput

import media.pixi.appkit.data.audio.Recording

interface TextInputDelegate {

    fun showOptions()
    fun hideOptions()
    fun onSendPressed(text: String)
    fun startTyping()
    fun sendAudio(recording: Recording)
    fun stopTyping()
    fun onKeyboardShow()
    fun onKeyboardHide()

}
